src/mono/browser/runtime/loader/run.ts

```diff
@@ -158,6 +158,10 @@ export class HostBuilder implements DotnetHostBuilder {
                 interpreterPgo: value,
                 interpreterPgoSaveDelay: autoSaveDelay
             });
+            if (monoConfig.runtimeOptions)
+                monoConfig.runtimeOptions.push("--interp-pgo-recording");
+            else
+                monoConfig.runtimeOptions = ["--interp-pgo-recording"];
```

Comment:
The idea around here is that `deep_merge_config` solves this type conditional assignement

```suggestion
                interpreterPgo: value,
                interpreterPgoSaveDelay: autoSaveDelay                
                runtimeOptions: ["--interp-pgo-recording"]
        }); 
```

Reply:
https://github.com/dotnet/runtime/blob/09b983ed0d04fd61ee35a739d080f4e8fc04de1b/src/mono/browser/runtime/loader/config.ts#L36

Reply:
I see, thanks. I'll make a follow-up PR to clean this up.